### PR TITLE
31 main view

### PR DESCRIPTION
**Pull request by BartoszBrodowski**
Adding main view. There's no callendar included as I was told not to add it yet. 
On Android looks like this:
MainView

### CODE REVIEW

**Review by komolcia — APPROVED** [review]
Everything looks fine. No changes required.

File: src/ui/components/MainView.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+				</View>
+				<Animated.FlatList
+					data={data}
+					contentContainerStyle={styles.popularItemsContainer}
+					horizontal={true}
+					renderItem={({ item }) => (
+						<View style={styles.popularItem}>
+							<Text>{item.name}</Text>
+						</View>
+					)}></Animated.FlatList>
```
Comment (kacperkruger) [inline src/ui/components/MainView.js:38]: You should use here self closing tag

File: src/ui/components/MainView.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	const data = [
+		{ name: '1', key: '1' },
+		{ name: '2', key: '2' },
+		{ name: '3', key: '3' },
+		{ name: '4', key: '4' },
+		{ name: '5', key: '5' },
+	];
+	return (
+		<SafeAreaView>
+			<StatusBar backgroundColor={'transparent'} translucent={false} />
```
Comment (kacperkruger) [inline src/ui/components/MainView.js:16]: I think status bar should be handled in app.js, so the status bar will be the same for every component

**Review by kacperkruger — CHANGES_REQUESTED** [review]
I think numbers in rectangles should be bigger and white like on figma

**Review by kacperkruger — COMMENTED** [review]
👍👍

**Review by kacperkruger — APPROVED** [review]

**Pull request merged**